subjects/prime_checker/README.md

```diff
@@ -17,7 +19,7 @@ pub enum PrimeErr {
     Divider(u32),
 }
 
-pub fn prime_checker(nb: u32) -> Option<Result<u32, PrimeErr>> {}
+pub fn prime_checker(nb: u32) -> Option<...> {}
```

Comment:
What do you think if we keep also the `Option` out of the picture?
Maybe using comments to avoid confusion:
```rust
pub fn prime_checker(nb: u32) -> /* Implement return type */ {}
```